src/framework/mlt_property.c

```diff
@@ -1213,6 +1215,7 @@ static inline double catmull_rom_interpolate(double y0, double y1, double y2, do
 
 /** Interpolate a new property value given a set of other properties.
  *
+ * \deprecated
```

Comment:
I propose to deprecate this function and keep the new function private in mlt_animation so that we can change it if necessary. I doubt any applications are using it. It just needed to be public so that mlt_animation could call it.